examples/driver.ts

```diff
@@ -1,5 +1,6 @@
 // yarn ts-node examples/driver.ts
 import { driver, util } from '../index';
+import { UnsupportedFeatureBehaviour } from '../src/driver';
```

Comment:
were you planning to use this for testing?

Reply:
Obviously I forgot to push my latest local changes, which include the test...